2025-02-26 20:51:32,802 INFO: 请求: GET http://192.168.72.128:5000/blog/login [in /data/myblog/app/__init__.py:141]
2025-02-26 20:51:32,831 INFO: 请求: GET http://192.168.72.128:5000/static/css/blog.css [in /data/myblog/app/__init__.py:141]
2025-02-26 20:51:32,846 INFO: 请求: GET http://192.168.72.128:5000/static/images/favicon.ico [in /data/myblog/app/__init__.py:141]
2025-02-26 20:51:38,184 INFO: 请求: POST http://192.168.72.128:5000/blog/login [in /data/myblog/app/__init__.py:141]
2025-02-26 20:51:38,303 INFO: 请求: GET http://192.168.72.128:5000/auth/login [in /data/myblog/app/__init__.py:141]
2025-02-26 20:51:38,357 INFO: 请求: GET http://192.168.72.128:5000/favicon.ico [in /data/myblog/app/__init__.py:141]
2025-02-26 20:51:38,358 WARNING: 页面未找到: http://192.168.72.128:5000/favicon.ico [in /data/myblog/app/__init__.py:109]
2025-02-26 20:51:54,687 INFO: 请求: POST http://192.168.72.128:5000/auth/login [in /data/myblog/app/__init__.py:141]
2025-02-26 20:51:54,815 INFO: 请求: GET http://192.168.72.128:5000/auth/login [in /data/myblog/app/__init__.py:141]
2025-02-26 20:51:54,876 INFO: 请求: GET http://192.168.72.128:5000/favicon.ico [in /data/myblog/app/__init__.py:141]
2025-02-26 20:51:54,877 WARNING: 页面未找到: http://192.168.72.128:5000/favicon.ico [in /data/myblog/app/__init__.py:109]
2025-02-26 20:52:33,614 INFO: MyBlog 启动 [in /data/myblog/app/__init__.py:69]
2025-02-26 20:52:33,614 INFO: 配置模式: development [in /data/myblog/app/__init__.py:70]
2025-02-26 20:52:33,614 INFO: 调试模式: True [in /data/myblog/app/__init__.py:71]
2025-02-26 20:52:33,615 INFO: 日志文件: /data/myblog/logs/myblog.log [in /data/myblog/app/__init__.py:72]
2025-02-26 20:52:33,634 INFO: 扩展初始化完成 [in /data/myblog/app/__init__.py:85]
2025-02-26 20:52:33,634 INFO: 数据库迁移初始化完成 [in /data/myblog/app/__init__.py:89]
2025-02-26 20:52:33,650 INFO: 蓝图注册完成 [in /data/myblog/app/__init__.py:96]
2025-02-26 20:52:33,651 INFO: 过滤器初始化完成 [in /data/myblog/app/__init__.py:115]
2025-02-26 20:52:33,652 INFO: 应用初始化完成 [in /data/myblog/app/__init__.py:166]
2025-02-26 20:52:34,231 INFO: MyBlog 启动 [in /data/myblog/app/__init__.py:69]
2025-02-26 20:52:34,231 INFO: 配置模式: development [in /data/myblog/app/__init__.py:70]
2025-02-26 20:52:34,231 INFO: 调试模式: True [in /data/myblog/app/__init__.py:71]
2025-02-26 20:52:34,231 INFO: 日志文件: /data/myblog/logs/myblog.log [in /data/myblog/app/__init__.py:72]
2025-02-26 20:52:34,246 INFO: 扩展初始化完成 [in /data/myblog/app/__init__.py:85]
2025-02-26 20:52:34,247 INFO: 数据库迁移初始化完成 [in /data/myblog/app/__init__.py:89]
2025-02-26 20:52:34,262 INFO: 蓝图注册完成 [in /data/myblog/app/__init__.py:96]
2025-02-26 20:52:34,262 INFO: 过滤器初始化完成 [in /data/myblog/app/__init__.py:115]
2025-02-26 20:52:34,263 INFO: 应用初始化完成 [in /data/myblog/app/__init__.py:166]
2025-02-26 20:52:44,155 INFO: 请求: GET http://192.168.72.128:5000/blog/login [in /data/myblog/app/__init__.py:141]
2025-02-26 20:52:44,156 INFO: Locating template 'blog/login.html':
    1: trying loader of application 'app'
       class: jinja2.loaders.FileSystemLoader
       encoding: 'utf-8'
       followlinks: False
       searchpath:
         - /data/myblog/app/templates
       -> found ('/data/myblog/app/templates/blog/login.html') [in /home/ops/.local/lib/python3.12/site-packages/flask/debughelpers.py:178]
2025-02-26 20:52:44,162 INFO: Locating template 'blog/base.html':
    1: trying loader of application 'app'
       class: jinja2.loaders.FileSystemLoader
       encoding: 'utf-8'
       followlinks: False
       searchpath:
         - /data/myblog/app/templates
       -> found ('/data/myblog/app/templates/blog/base.html') [in /home/ops/.local/lib/python3.12/site-packages/flask/debughelpers.py:178]
2025-02-26 20:52:44,209 INFO: 请求: GET http://192.168.72.128:5000/static/css/blog.css [in /data/myblog/app/__init__.py:141]
2025-02-26 20:52:44,230 INFO: 请求: GET http://192.168.72.128:5000/static/images/favicon.ico [in /data/myblog/app/__init__.py:141]
2025-02-26 20:53:00,514 INFO: 请求: POST http://192.168.72.128:5000/blog/login [in /data/myblog/app/__init__.py:141]
2025-02-26 20:53:00,662 INFO: 请求: GET http://192.168.72.128:5000/static/css/blog.css [in /data/myblog/app/__init__.py:141]
2025-02-26 20:53:00,681 INFO: 请求: GET http://192.168.72.128:5000/static/images/favicon.ico [in /data/myblog/app/__init__.py:141]
2025-02-26 20:53:06,495 INFO: 请求: POST http://192.168.72.128:5000/blog/login [in /data/myblog/app/__init__.py:141]
2025-02-26 20:53:06,631 INFO: 请求: GET http://192.168.72.128:5000/blog/ [in /data/myblog/app/__init__.py:141]
2025-02-26 20:53:06,649 INFO: Locating template 'blog/index.html':
    1: trying loader of application 'app'
       class: jinja2.loaders.FileSystemLoader
       encoding: 'utf-8'
       followlinks: False
       searchpath:
         - /data/myblog/app/templates
       -> found ('/data/myblog/app/templates/blog/index.html') [in /home/ops/.local/lib/python3.12/site-packages/flask/debughelpers.py:178]
2025-02-26 20:53:06,660 INFO: Locating template 'blog/layout.html':
    1: trying loader of application 'app'
       class: jinja2.loaders.FileSystemLoader
       encoding: 'utf-8'
       followlinks: False
       searchpath:
         - /data/myblog/app/templates
       -> found ('/data/myblog/app/templates/blog/layout.html') [in /home/ops/.local/lib/python3.12/site-packages/flask/debughelpers.py:178]
2025-02-26 20:53:06,699 INFO: 请求: GET http://192.168.72.128:5000/static/css/blog.css [in /data/myblog/app/__init__.py:141]
2025-02-26 20:53:06,716 INFO: 请求: GET http://192.168.72.128:5000/favicon.ico [in /data/myblog/app/__init__.py:141]
2025-02-26 20:53:06,717 WARNING: 页面未找到: http://192.168.72.128:5000/favicon.ico [in /data/myblog/app/__init__.py:109]
2025-02-26 20:53:06,718 INFO: Locating template 'errors/404.html':
    1: trying loader of application 'app'
       class: jinja2.loaders.FileSystemLoader
       encoding: 'utf-8'
       followlinks: False
       searchpath:
         - /data/myblog/app/templates
       -> found ('/data/myblog/app/templates/errors/404.html') [in /home/ops/.local/lib/python3.12/site-packages/flask/debughelpers.py:178]
2025-02-26 20:53:22,385 INFO: 请求: GET http://192.168.72.128:5000/admin/ [in /data/myblog/app/__init__.py:141]
2025-02-26 20:53:22,387 INFO: Locating template 'admin/index.html':
    1: trying loader of application 'app'
       class: jinja2.loaders.FileSystemLoader
       encoding: 'utf-8'
       followlinks: False
       searchpath:
         - /data/myblog/app/templates
       -> found ('/data/myblog/app/templates/admin/index.html') [in /home/ops/.local/lib/python3.12/site-packages/flask/debughelpers.py:178]
2025-02-26 20:53:22,389 INFO: Locating template 'admin/layout.html':
    1: trying loader of application 'app'
       class: jinja2.loaders.FileSystemLoader
       encoding: 'utf-8'
       followlinks: False
       searchpath:
         - /data/myblog/app/templates
       -> found ('/data/myblog/app/templates/admin/layout.html') [in /home/ops/.local/lib/python3.12/site-packages/flask/debughelpers.py:178]
2025-02-26 20:53:22,438 INFO: 请求: GET http://192.168.72.128:5000/favicon.ico [in /data/myblog/app/__init__.py:141]
2025-02-26 20:53:22,439 WARNING: 页面未找到: http://192.168.72.128:5000/favicon.ico [in /data/myblog/app/__init__.py:109]
2025-02-26 20:53:28,346 INFO: 请求: GET http://192.168.72.128:5000/admin/users/ [in /data/myblog/app/__init__.py:141]
2025-02-26 20:53:28,348 WARNING: Forbidden: 403 Forbidden: 您没有管理员权限，无法访问此页面 [in /data/myblog/app/extensions.py:138]
2025-02-26 20:53:28,349 INFO: Locating template 'errors/403.html':
    1: trying loader of application 'app'
       class: jinja2.loaders.FileSystemLoader
       encoding: 'utf-8'
       followlinks: False
       searchpath:
         - /data/myblog/app/templates
       -> found ('/data/myblog/app/templates/errors/403.html') [in /home/ops/.local/lib/python3.12/site-packages/flask/debughelpers.py:178]
2025-02-26 20:53:28,376 INFO: 请求: GET http://192.168.72.128:5000/static/css/blog.css [in /data/myblog/app/__init__.py:141]
2025-02-26 20:53:28,392 INFO: 请求: GET http://192.168.72.128:5000/static/images/favicon.ico [in /data/myblog/app/__init__.py:141]
2025-02-26 20:53:30,104 INFO: 请求: GET http://192.168.72.128:5000/admin/ [in /data/myblog/app/__init__.py:141]
2025-02-26 20:53:30,152 INFO: 请求: GET http://192.168.72.128:5000/favicon.ico [in /data/myblog/app/__init__.py:141]
2025-02-26 20:53:30,152 WARNING: 页面未找到: http://192.168.72.128:5000/favicon.ico [in /data/myblog/app/__init__.py:109]
2025-02-26 20:53:33,082 INFO: 请求: POST http://192.168.72.128:5000/auth/logout [in /data/myblog/app/__init__.py:141]
2025-02-26 20:53:33,084 INFO: 用户 test 已登出 [in /data/myblog/app/controllers/auth/__init__.py:64]
2025-02-26 20:53:33,106 INFO: 请求: GET http://192.168.72.128:5000/auth/login [in /data/myblog/app/__init__.py:141]
2025-02-26 20:53:33,107 INFO: Locating template 'auth/login.html':
    1: trying loader of application 'app'
       class: jinja2.loaders.FileSystemLoader
       encoding: 'utf-8'
       followlinks: False
       searchpath:
         - /data/myblog/app/templates
       -> found ('/data/myblog/app/templates/auth/login.html') [in /home/ops/.local/lib/python3.12/site-packages/flask/debughelpers.py:178]
2025-02-26 20:53:33,176 INFO: 请求: GET http://192.168.72.128:5000/favicon.ico [in /data/myblog/app/__init__.py:141]
2025-02-26 20:53:33,177 WARNING: 页面未找到: http://192.168.72.128:5000/favicon.ico [in /data/myblog/app/__init__.py:109]
2025-02-26 20:53:40,578 INFO: 请求: POST http://192.168.72.128:5000/auth/login [in /data/myblog/app/__init__.py:141]
2025-02-26 20:53:40,701 INFO: 请求: GET http://192.168.72.128:5000/auth/login [in /data/myblog/app/__init__.py:141]
2025-02-26 20:53:40,751 INFO: 请求: GET http://192.168.72.128:5000/favicon.ico [in /data/myblog/app/__init__.py:141]
